tasks/install.yml

```diff
@@ -18,14 +18,15 @@
     - "{{ ntfy_config_dir_path }}"
     - "{{ ntfy_data_path }}"
 
-- name: Ensure ntfy path for attachment cache directory exists
+- name: Ensure ntfy paths for cache directories exists
```

Comment:
It's better to merge this task with the directory-creating task above, like we do for all other roles.

This is more consistent and also offers a slight performance benefit (each task causes additional overhead).

Reply:
I've mistakenly separated them as installation failed due to my incorrect settings. I do not seem to know what I am doing!